tasks/eth/027-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md

```diff
@@ -0,0 +1,140 @@
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+- **Before**: `0x0000000000000000000000000000000000000000000000000000000001c9c380`
+- **After**: `0x0000000000000000000000000000000000095506000186a00000000001c9c380`
+- **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `100000` (`cast td 0x000186a0`) and `611590` (`cast td 0x00095506`) respectively. These share a slot with the `gasLimit` which remans at `30000000` (`cast td 0x0000000001c9c380`).
+
+and
+
+- **Key**: `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+- **Before**: `0x000000000000000000000000ba2492e52f45651b60b8b38d4ea5e2390c64ffb1`
+- **After**: `0x000000000000000000000000ab9d6cb7a427c0765163a7f45bb91cafe5f2d375`
+- **Meaning**: Updates the implementation address of the Proxy to the standard SystemConfig implementation.
+
+### `0x34A564BbD863C4bf73Eca711Cf38a77C4Ccbdd6A` (`SystemConfigProxy` for arena-z-mainnet)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000066`
+- **Before**: `0x00000000000000000000000000000000000000000000000000000000000a6fe0`
+- **After**: `0x01000000000000000000000000000000000000000000000000000000000a6fe0`
+- **Meaning**: Updates the `scalar` storage variable to reflect a scalar version of `1`.
+
+and
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+- **Before**: `0x0000000000000000000000000000000000000000000000000000000003938700`
+- **After**: `0x0000000000000000000000000000000000000000000c3c9d0000000003938700`
+- **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `801949` (`cast td 0x000c3c9d`) and `0` respectively. These share a slot with the `gasLimit` which remains at `60000000` (`cast td 0x0000000003938700`).
+
+and
+
+- **Key**: `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+- **Before**: `0x000000000000000000000000f56d96b2535b932656d3c04ebf51babff241d886`
+- **After**: `0x000000000000000000000000ab9d6cb7a427c0765163a7f45bb91cafe5f2d375`
+- **Meaning**: Updates the implementation address of the Proxy to the standard SystemConfig implementation.
+
+
+### `0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A` (`ProxyAdminOwner` for all chains in this task)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000005`
+- **Before**: `0x0000000000000000000000000000000000000000000000000000000000000008`
+- **After**: `0x0000000000000000000000000000000000000000000000000000000000000009`
+- **Meaning**: Nonce increments by 1
+
+and
+
+- **Key**: In the [`approvedHashes` section](#nested-safe-state-overrides-and-changes)
+- **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000`
+- **After**: `0x0000000000000000000000000000000000000000000000000000000000000001`
+- **Meaning**: approvedHashes update. See above.
+
+
+### Other nonce changes
```

Comment:
Because, of the recent nonce issue we should make sure that all the nonces are viewable here (to reduce the friction with the signers).

Reply:
As you wish [094c5aa](https://github.com/ethereum-optimism/superchain-ops/pull/454/commits/094c5aa22243a26449d12237ee70bfcf32c4c19a)